.nvmrc

```diff
@@ -1 +1 @@
-18.20
+v18.20.4
```

Comment:
Building the web image will fail. 
> REGISTRY=apache VERSION=latest make clean image

The reason is the unexpected 'v' in the resolved node tag name in Dockerfile.
Error Message:
`ERROR: failed to solve: node:v18.20.4-alpine: docker.io/library/node:v18.20.4-alpine: not found`

Reply:
Thanks. I had to regenerate `pnpm-lock.yaml` as well.